package.json

```diff
@@ -1,24 +1,18 @@
 {
-  "name": "redux-immutable-state-invariant",
-  "version": "1.2.0",
-  "description": "Redux middleware that detects mutations between and outside redux dispatches. For development use only.",
+  "name": "object-invariant-test-helper",
+  "version": "1.0.0",
   "main": "dist/index.js",
   "scripts": {
     "clean": "rm -rf dist",
     "build": "babel --out-dir dist src",
     "test": "mocha --compilers js:babel/register",
-    "prepublish": "npm run clean && npm run build",
-    "example": "cd example && rm -rf node_modules/redux-immutable-state-invariant && npm install && npm start"
+    "prepublish": "npm run clean && npm run build"
   },
-  "author": "Leonardo Andres Garcia Crespo <leoasis@gmail.com>",
+  "author": "Jana Beck <jana@tidepool.org>",
```

Comment:
Hmm, I think npm has a ["contributors" alternative](https://docs.npmjs.com/files/package.json#people-fields-author-contributors), maybe we should use that to include original author? And add you if you like ;)